bitnami/redis/values.yaml

```diff
@@ -921,6 +930,15 @@ replica:
     ## NOTE: requires replica.persistence.enabled: true
     ##
     existingClaim: ""
+  ## persistentVolumeClaimRetentionPolicy
```

Comment:
Please apply the formatting correction to this section as well.

Reply:
@fevisera Formatting applied.